homeassistant/components/deconz/config_flow.py

```diff
@@ -41,6 +43,10 @@ def __init__(self):
         self.bridges = []
         self.deconz_config = {}
 
+    async def async_step_init(self, user_input=None):
```

Comment:
Where is this called?

Reply:
IIRC

        return self.async_show_form(
            step_id='init',
            data_schema=vol.Schema({
                vol.Required(CONF_HOST): str,
                vol.Required(CONF_PORT, default=DEFAULT_PORT): int,
            }),
        )

Reply:
Which we keep to not break translations

Reply:
Then add a comment

Reply:
I will clarify the comment